### PR TITLE
[DO NOT MERGE YET] content_cards_android

### PR DESCRIPTION
**Pull request by dkim32**
# Pull Request/Issue Resolution

**Description of Change:**
Revising hidden docs for content cards: 
https://www.braze.com/docs/content_cards_android/

New sections: 
- Overview
- Customization 
- Read & Unread indicators
- Refreshing the feed
- Badges 
- Data models

**Reason for Change:**
GA launch for feature. 

Closes #**ISSUE_NUMBER_HERE**

---
---

## PR Checklist
- [ ] Ensure you have completed our CLA.
- [ ] Tag @EmilyNecciai as a reviewer when the your work is done and ready to be reviewed for merge. 
- [ ] Consult the [Docs Text Formatting Guide](https://github.com/Appboy/success/wiki/Docs-Text-Formatting-Guide) to be sure you're utilizing the proper markdown formatting.
- [ ] Consult the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/success/wiki/Writing-Style-Guide-&-Best-Practices) to be sure you're aligning with our voice and other style best practices.
- [ ] [Preview your deployed changes](https://homeslice.braze.com/docs) to confirm that none of your changes break production. Pay close attention to links and images.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

---
---

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/read_unread_indicators.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+page_order: 2
+search_rank: 5
+platform: Android
+---
+
+## Read/Unread Indicators {#read-indicators-for-android}
+
+Braze allows you to optionally toggle on an Unread/Read indicator on Content Cards.
+
+Unread content cards pic here
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/read_unread_indicators.md]: remove

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/data_models.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+---
+nav_title: Data Models
+page_order: 5
+search_rank: 5
+platform: Android
+---
+
+## Card Types {#card-types-for-android}
+Braze has 5 unique Content Cards card types which share a base model. Each card type also has additional card-specific properties which are listed below.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/data_models.md]: only four

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+## Key-Value Pairs
+`Card` objects may optionally carry key-value pairs as `extras`. These can be used to send data down along with a `Card` for further handling by the application.
+
+See the [Javadoc][36] for more information.
+
+## GIFs {#gifs-news-content-cards}
+
+{% include archive/android/gifs.md channel="Content Cards" %}
+
+[1]:{% image_buster /assets/img_archive/contentcard.png %}
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md]: i think its this image should be replaced - the phone graphic and content card UI are outdated

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/badges.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+---
+## Adding a Badge
+
+You can request the number of unread cards at any time by calling:
+
+{% tabs %}
+{% tab JAVA %}
+
+```java
+Appboy.getInstance(context).getContentCardUnviewedCount()
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/badges.md]: `Appboy.getInstance(context).getContentCardUnviewedCount();`

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/refreshing_the_feed.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+---
+## Refreshing Content Cards
+
+You can queue a manual refresh of the Braze Content Cards at any time by calling:
+
+{% tabs %}
+{% tab JAVA %}
+
+```java
+Appboy.getInstance(context).requestContentCardsRefresh()
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/refreshing_the_feed.md]: `Appboy.getInstance(context).requestContentCardsRefresh();`

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/data_models.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+|Property | Description |
+|---|---|
+|`getImageUrl()` | Returns the URL of the card’s image.|
+|`getTitle()` | Returns the title text for the card.
+|`getDescription()` | Returns the body text for the card.
+|`getUrl()` | Returns the URL that will be opened after the card is clicked. It can be a http(s) URL or a protocol URL.
+|`getDomain()` | Returns the link text for the property URL.
+
+#### Text Announcement Card (Captioned Image without Image) {#text-Announcement-card-for-android}
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/data_models.md]: "Classic"

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/data_models.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+[Text Announcement Cards][32] are clickable cards containing descriptive text. In addition to the base card properties:
+
+|Property | Description |
+|---|---|
+|`getTitle()` | Returns the title text for the card.
+|`getDescription()` | Returns the body text for the card.
+|`getUrl()` | Returns the URL that will be opened after the card is clicked. It can be a http(s) URL or a protocol URL.
+|`getDomain()` | Returns the link text for the property URL.
+
+#### Short News Card
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/data_models.md]: remove

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/data_models.md (hunk 2 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/android/content_cards/data_models.md]: actually only three :P classic, captioned image, and banner

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/read_unread_indicators.md (hunk 1 above)
Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/content_cards/read_unread_indicators.md]: or add image

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/badges.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+{% endtab %}
+{% tab KOTLIN %}
+
+```kotlin
+Appboy.getInstance(context).contentCardUnviewedCount
+```
+
+{% endtab %}
+{% endtabs %}
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/badges.md]: I'd link to https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/Appboy.html#getContentCardUnviewedCount-- somewhere in here

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/overview.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
+---
```
[the remark this answers is not in the document]
Comment (dkim32) [reply _docs/_developer_guide/platform_integration_guides/android/content_cards/overview.md]: fixed. ty!

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md [hunk 10]
```diff
@@ -0,0 +1,403 @@
...
+platform: Android
+---
+## Content Cards Customization {#content-cards-customization-for-android}
+
+### Default Styling {#default-styling-for-android}
+
+The Braze UI elements (IAMs and Content Cards) come with a default look and feel that matches the Android standard UI guidelines and provides a seamless experience. You can see these default styles in the `res/values/style.xml` file in the Braze SDK distribution.
+
+```xml
+  <style name="Appboy"/>
```
[the remark this answers is not in the document]
Comment (dkim32) [reply _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md]: @Bucimis @radixdev just remove `  <style name="Appboy"/> `?

[the remark this answers is not in the document]
Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md]: yeah. I think the section would be more concise as a result.

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md (hunk 3 above)
Comment (dkim32) [reply _docs/_developer_guide/platform_integration_guides/android/content_cards/customization.md]: removing. not included within the documentation.

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/overview.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+Braze recommends that customers who use our News Feed tool move over to our Content Cards messaging channel - it is more flexible, customizable, and reliable. It is also easier to find and use in the Braze product. Contact your Braze account manager for more information.
+{% endalert %}
+
+## When to Use Content Cards
+
+ Content Cards typically sit in a feed of sorts (but not necessarily), and help you take advantage of the visual space by incorporating images and graphics that stand out. You can personalize the cards based on user actions, onboard your customers with a checklist, and much more!
+
+### Great Use Cases
+
+ - Showcase new content.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/overview.md]: extra space

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/data_models.md [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+### Classic Card {#text-Announcement-card-for-android}
+[Text Announcement Cards][32] are clickable cards containing descriptive text. In addition to the base card properties:
+
+|Property | Description |
+|---|---|
+|`getTitle()` | Returns the title text for the card.
+|`getDescription()` | Returns the body text for the card.
+|`getUrl()` | Returns the URL that will be opened after the card is clicked. It can be a http(s) URL or a protocol URL.
+|`getDomain()` | Returns the link text for the property URL.
+
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/data_models.md]: whitespace

File: _docs/_developer_guide/platform_integration_guides/android/content_cards/overview.md [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+## Content Cards & Feed
+
+ This is what it looks like for your users to open a basic Content Card feed. As you can see, there are three basic types of cards that can sit in the feed - a Banner Card, a Captioned Content Card, and a Classic Content Card.
+
+ ![Content Cards Feed]({% image_buster /assets/img/sample-torchy-feed-content-cards-braze.png %}){: height="50%" width="50%"}
+
+## Content Cards Integration Overview {#content-cards-integration-for-android}
+
+ In Android, Content Cards and the feedback form are implemented as [Fragments][2] that are available in the Braze Android UI project. View [Google's documentation on Fragments][3] for information on how to add a Fragment to an Activity.
+
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/content_cards/overview.md]: ContentCards -> Content Cards feed
set -> created?

**Pull request merged**